### PR TITLE
Add openshift/cluster-openshift-controller-manager-operator

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @deads2k

### CODE REVIEW

File: ci-operator/config/openshift/cluster-openshift-controller-manager-operator/master.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  from: bin
+  commands: |
+    go get -u golang.org/x/tools/imports
+    make verify -k
+
+- as: unit
+  from: test-bin
+  commands: |
+    mkdir -p /tmp/artifacts
+    go get -u github.com/openshift/release/tools/gotest2junit
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cluster-openshift-controller-manager-operator/master.yaml]: We strongly suggest you don't chain command here and instead do it in your repo

File: ci-operator/config/openshift/cluster-openshift-controller-manager-operator/master.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+
+tests:
+- as: verify
+  from: bin
+  commands: |
+    go get -u golang.org/x/tools/imports
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cluster-openshift-controller-manager-operator/master.yaml]: ditto here

**Comment by tnozicka** [comment]
/assign @smarterclayton 
(this need repo setup and be mirrored)

**Pull request merged**